MapboxNavigation/RouteTableViewController.swift

```diff
@@ -72,14 +72,14 @@ extension RouteTableViewController: UITableViewDelegate, UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return routeController.routeProgress.currentLeg.steps.count
+        return routeController.routeProgress.allSteps.count
```

Comment:
Instead of flattening the list of steps, we should have multiple sections (instead of just one), each section listing the steps for a single leg.